pkg/semrel/semrel.go

```diff
@@ -66,10 +70,21 @@ func applyChange(rawVersion string, rawChange *Change, allowInitialDevelopmentVe
 	} else {
 		preRel += ".1"
 	}
-	newVersion, _ = version.SetPrerelease(preRel)
+
+	newVersion = must(version.SetPrerelease(preRel))
+	newVersion = must(newVersion.SetMetadata(version.Metadata()))
```

Comment:
The way this produces useful behavior is ... not immediately obvious until I re-read the slack thread, and even then is a little confusing -- some comments or other documentation would help.